shimmingtoolbox/cli/b0shim.py

```diff
@@ -978,19 +979,31 @@ def _get_current_shim_settings(json_data):
 @timeit
 def _plot_coefs(coil, slices, static_coefs, path_output, coil_number, rt_coefs=None, pres_probe_min=None,
                 pres_probe_max=None, units='', bounds=None):
-    n_shims = static_coefs.shape[0]
-    fig = Figure(figsize=(8, 4 * n_shims), tight_layout=True)
+    # We want to find how much slices are not shimmed to have a smaller file and reduced the plotting and saving time
+    shimmed_slice = []
+    n_shims = len(slices)
+    list_slice_wo_shimmed = ""
+    for i_shim in range(n_shims):
+        if np.any(static_coefs[i_shim]):
+            shimmed_slice.append(i_shim)
+        else:
+            # Get a string with the number of all the unshimmed slices
+            list_slice_wo_shimmed = list_slice_wo_shimmed + str(slices[i_shim]) + ","
+            # Get the last index where the shimmed correction is null will permit to plot the last one as an example
+            # for all the other one
```

Comment:
```suggestion
            # Get the last index where the shimmed correction is null, this will allow to plot the last one as an example
            # for all the other ones
```